libs/openchallenges/ui/src/lib/paginator/paginator.component.ts

```diff
@@ -9,22 +18,28 @@ import { PaginatorModule as PrimengPaginatorModule } from 'primeng/paginator';
   templateUrl: './paginator.component.html',
   styleUrls: ['./paginator.component.scss'],
 })
-export class PaginatorComponent implements OnInit {
+export class PaginatorComponent {
   @Input({ required: true }) pageNumber = 0; // index of the new page
   @Input({ required: false }) pageLinkSize = 5;
   @Input({ required: true }) pageSize = 0; // number of rows to display in new page
   @Input({ required: true }) totalRecords = 0;
```

Comment:
That that correspond to the `totalElements` included with a page metadata returned by the backend? If yes, consider using the same name.